### DISCUSSION TITLE
window control: min/max size, close

### DISCUSSION THREAD

**Discussion by akvadrako** [Igalia/cog#289] [Q&A]
Hello. I am exploring the idea of using cog as the base for an electron alternative.

Is it possible to control the window size (including min/max) and to close it? If not, is the a feature the project would be interested to have?

**Comment by philn** [comment]
Maybe join efforts with [Electroff](https://webreflection.medium.com/electroff-an-electron-less-alternative-10920c6003b8) https://github.com/WebReflection/electroff

**Reply by akvadrako** [reply]
Indeed that might be useful for the backend component.
